R/translate-sql-helpers.r

```diff
@@ -170,3 +170,22 @@ sql_cast <- function(type) {
     build_sql("CAST(", x, " AS ", sql(type), ")")
   }
 }
+
+#' @rdname sql_variant
+#' @export
+sql_formula_log <- function(f, base = exp(1)) {
```

Comment:
I think these can just be `sql_log` and `sql_cot`, and like `sql_cast()` etc should return functions